src/components/raspberry-pi/image-block/_index.scss

```diff
@@ -106,6 +106,35 @@
   }
 }
 
+.sc-rp-image-block--padded-image {
+  .sc-rp-image-block__image-container {
+    @include padding.inline('bee');
+    transform: translateX(0);
+    width: auto;
+  }
+
+  @include device.min-width(760px) {
+    .sc-rp-image-block__image-container {
+      @include padding.inline('none');
```

Comment:
Aaaaaand this is why we need to do something about `@include padding` and `@include margin`. It's not obvious that these mixins actually generate media queries of their own, so now we're have media queries in media queries, which will result in styles that can NEVER be actually applied included in the output.

This is entirely on me, and Greg and I will be working to fix this in short order with a new concept for `vspace` and `hspace`, which will use CSS clamp under the hood.

I'm not sure what to do here. If this was an application I'd be like "fine let's ship it", but since this is Sauce itself I feel that it should be better about this kind of thing.

**This is entirely my fault**, so I'll happily be the one that has to sort this out. What do you reckon?

Reply:
> which will result in styles that can NEVER be actually applied included in the output.

It does work fwiw. Also, I was aware that they generate media queries since the actual rem value changes depending on browser width. I don't know how else we'd handle this though, if we need to change padding/margin levels as we reflow components, then what other choice do we have?